stdlib/jwt/src/main/ballerina/src/jwt/inbound_jwt_auth_provider.bal

```diff
@@ -38,10 +38,14 @@ public type InboundJwtAuthProvider object {
         self.inboundJwtCache = jwtValidatorConfig.jwtCache;
     }
 
-    # Authenticate with a JWT token.
-    #
-    # + credential - Jwt token extracted from the authentication header
-    # + return - `true` if authentication is successful, othewise `false` or `auth:Error` occurred during JWT validation
+# Authenticates with a JWT.
+#```ballerina
+# boolean|auth:Error result = authenticate(string credential);
+# ```
+#
+# + credential - JWT extracted from the authentication header
+# + return - `true` if authentication is successful, othewise `false` or else an `auth:Error` if JWT validation
```

Comment:
```suggestion
# + return - `true` if authentication is successful, othewise `false` or else an `auth:Error` if JWT validation
```
Please refer the template for the correct format of the return statement description.